### PR TITLE
Naruto Shippuden Ultimate Ninja 3: Probably a better fix for the video hang issue.

### LINKED ISSUE #9591: Naruto Shippuden Ultimate Ninja Heroes 3 : Black screen after title logo - Since 1.3-37-g8b5693b
On: Windows 10 64 bit & Android 7.1.1 ARM

Last work: 1.3-31-gb3a7c0e
First break: 1.3-37-g8b5693b
I try test on latest build agian (1.4-70) but not work fine.

Here log: https://drive.google.com/file/d/0B4Y2FBdaxHTDYTRoQ2Z2LWJpVGM/view?usp=drivesdk

### PR DESCRIPTION
**Pull request by hrydgard**
Fixes #9591, replaces #11772 

Needs testing with other games!

### CODE REVIEW

File: Core/HLE/sceMpeg.cpp [hunk 1]
```diff
@@ -1694,7 +1695,11 @@ static int sceMpegGetAtracAu(u32 mpeg, u32 streamId, u32 auAddr, u32 attrAddr)
 		Memory::Write_U32(0, attrAddr);
 	}
 
-	DEBUG_LOG(ME, "%x=sceMpegGetAtracAu(%08x, %08x, %08x, %08x)", result, mpeg, streamId, auAddr, attrAddr);
+	if (result == 0) {
+		DEBUG_LOG(ME, "%x=sceMpegGetAtracAu(%08x, %08x, %08x, %08x)", result, mpeg, streamId, auAddr, attrAddr);
```
Comment (unknownbrackets) [inline Core/HLE/sceMpeg.cpp]: Might be nice to change this to `hleLogSuccessI` / `hleLogWarning` while changing it...

-[Unknown]

**Review by unknownbrackets — COMMENTED** [review]
Need to test this on more games indeed, but seems right to me.

I'll note that the warnings do occur at various times in games normally.

-[Unknown]

File: Core/HLE/sceMpeg.cpp (hunk 1 above)
Comment (hrydgard) [reply Core/HLE/sceMpeg.cpp]: Since we're doing a hleDelayResult below, I'm gonna leave it as is for now. I'm switching back to DEBUG.

**Comment by hrydgard** [comment]
I've tested this with a bunch of games that use video, have not noticed any problems. Let's try it.

**Comment by Emulatorer** [comment]
I will test if NSUNH3 is really fixed ty 😊

**Comment by Emulatorer** [comment]
So it's just the same using the cheat code skip intro movie 😐

The intro video is gone 🤔

**Comment by hrydgard** [comment]
This certainly didn't intend to skip a video. If there's a video missing, that's another bug...

**Comment by Emulatorer** [comment]
Here's the video
https://youtu.be/JeYJgZq3BRk

**Comment by hrydgard** [comment]
Do you have a video where it shows the missing video (maybe from a real PSP)?

**Comment by Emulatorer** [comment]
Yeah you're right I thought a long movie intro I tested v1.3.0.1 they are the same introduction in ppsspp-v1.7.5-438-gccfcbc746 buildbot 😅

So finally the issue solved thanks for your hard work sir to fixed a long issue for this Naruto game in ppsspp

@hrydgard God bless you and thank you again

**Pull request merged**